main.go

```diff
@@ -462,11 +519,32 @@ func main() {
 	td := tdglib.NewToDoGenerator(env.sourceRoot(),
 		includePatterns,
 		excludePatterns,
-		false /*blame*/,
+		env.assignFromBlame,
 		env.minWords,
 		env.minChars,
 		env.concurrency)
 
+	if env.assignFromBlame {
+		root := env.sourceRoot()
+		for strings.HasSuffix(root, "/.") || strings.HasSuffix(root, "/") {
+			root = strings.TrimSuffix(root, "/.")
+			root = strings.TrimSuffix(root, "/")
+		}
```

Comment:
What is the purpose of `/.` logic? What kind of paths you're trying to exclude?

Reply:
The default `env.sourceRoot()` points to `/github/workspace/.`. I was playing arround with this git command and observed that it works only when the path points to the folder without a trailing `/` or `/.` - `/github/workspace`.

So, in my code here, while the path ends with `/.` or `/`, I keep trimming it. I know it would be rare that someone would inlcude a path like `/Folder/Subfolder/././`, but this code would handle cases like that and change it to `/Folder/Subfolder`.